openfecli/tests/commands/test_gather.py

```diff
@@ -4,79 +4,105 @@
 import tarfile
 import pytest
 
-from openfecli.commands.gather import gather
+from openfecli.commands.gather import (
+    gather, format_estimate_uncertainty, _get_column
+)
+
+@pytest.mark.parametrize('est,unc,unc_prec,est_str,unc_str', [
+    (12.432, 0.111, 2, "12.43", "0.11"),
+    (0.9999, 0.01, 2, "1.000", "0.010"),
+    (1234, 100, 2, "1230", "100"),
+])
+def test_format_estimate_uncertainty(est, unc, unc_prec, est_str, unc_str):
+    assert format_estimate_uncertainty(est, unc, unc_prec) == (est_str, unc_str)
+
+@pytest.mark.parametrize('val, col', [
+    (1.0, 1), (0.1, -1), (-0.0, 0), (0.0, 0), (0.2, -1), (0.9, -1),
+    (0.011, -2), (9, 1), (10, 2), (15, 2),
+])
+def test_get_column(val, col):
+    assert _get_column(val) == col
 
 
 @pytest.fixture
 def results_dir(tmpdir):
     with tmpdir.as_cwd():
-        with resources.path('openfecli.tests.data', 'results.tar.gz') as f:
+        with resources.path('openfecli.tests.data', 'rbfe_results.tar.gz') as f:
             t = tarfile.open(f, mode='r')
             t.extractall('.')
 
         yield
 
+_EXPECTED_DG = b"""
+ligand	DG(MLE) (kcal/mol)	uncertainty (kcal/mol)
+lig_ejm_31	-0.09	0.05
+lig_ejm_42	0.7	0.1
```

Comment:
the sig fig here seems to be inconsistent.  i.e. `0.7` != `0.70`

Reply:
Implementation according to https://github.com/OpenFreeEnergy/openfe/pull/502#discussion_r1273261215, which I think is the right was to handle this. We don't have a fixed "sig figs of our estimate," so we say "well, we're correct up to the first digit of error, which will vary for each row of the table."

Assuming a fixed number of sig figs for the estimate gives rounded estimates outside the actual error bars, as described in https://github.com/OpenFreeEnergy/openfe/pull/502#issue-1819029716.